src/CattyTests/Parser & Serialization/CatrobatLanguage0.95/XMLParserTests/Unit Tests/XMLParserUserVariableTests095.swift

```diff
@@ -46,44 +46,44 @@ final class XMLParserUserVariableTests095: XMLAbstractTest {
 
         let project = self.parserContext?.parse(from: xmlRoot.rootElement(), withClass: Project.self) as! Project
         XCTAssertNotNil(project, "project is nil")
-        XCTAssertEqual(8, project.allObjects().count, "Invalid number of object variables")
+        XCTAssertEqual(8, project.scene.objects().count, "Invalid number of object variables")
```

Comment:
Please use `allObjects` method directly (also in subsequent calls of this test class).